ChangingArray.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW_4
+{
+    public class ChangingArray
+    {
+        public int LetterArray(int[] array, byte veribal)
+        {
+            Console.WriteLine();
+            Console.WriteLine(new string('-', 50));
+
+            string infoArray = veribal == 1 ? "Array of odd numbers in letters" : "Array of non-odd numbers in letters";
```

Comment:
Are there any other options among ever and odd? If no, byte can be replaced with bool trigger, as originally discussed